omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -45,13 +46,7 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
         }
 
     self.validateCondition = self.validateCondition || function () {
-        var concept ;
-        if($scope.concept.concept.uuid) {
-            concept = $scope.concept.concept;
-        } else {
-            concept = $scope.condition.concept;
-        }
-
+        var concept = $scope.concept.concept;
```

Comment:
I'm not sure if `$scope.concept.concep` will always store the value of concept, maybe it will be better to do something like that:
```suggestion
        var concept;
        if($scope.concept.concept) {	
            concept = $scope.concept.concept;	
        } else {	
            concept = $scope.condition.concept;	
        }
```